com.unity.render-pipelines.high-definition/Documentation~/Override-Exposure.md

```diff
@@ -65,6 +65,10 @@ To configure **Automatic Mode**, select the **Metering Mode**. This tells the Ca
 
 ![](Images/Override-Exposure3.png)
 
+- **Mask Weighted**: The Camera applies a weight to every pixel in the buffer and then uses them to measure the exposure. The weighting is specified by the texture provided in the **Weight Texture Mask** field. Note that if no texture is provided, then this metering mode is equivalent to Average.
```

Comment:
**Mask Weighted**: The Camera applies a weight to every pixel in the buffer then uses the weights to measure the exposure. To specify the weighting, this technique uses the Texture set in the **Weight Texture Mask** field. Note that, if you do not provide a Texture, this metering mode is equivalent to **Average**.